src/com/google/javascript/jscomp/DiagnosticGroups.java

```diff
@@ -149,8 +150,9 @@ public DiagnosticGroup forName(String name) {
           + "underscore, "
           + "visibility";
 
-  public static final DiagnosticGroup COMMON_JS_MODULE_LOAD =
-      DiagnosticGroups.registerGroup("commonJsModuleLoad",
```

Comment:
If anyone already has `@suppress {commonJsModuleLoad}` in their code then they'll have to change it, right? We should mention that in the release notes when we bring this in.

Reply:
I did consider that - just didn't like even more groups.

Alternatively I can add a new one and remove the commonJs specific one from the advertised list.

Reply:
Sounds good, we can just note it in the release notes.